contracts/weapons.sol

```diff
@@ -95,8 +89,6 @@ contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable,
     int128 public powerMultPerPointPWR; // 0.2575% (+3%)
     int128 public powerMultPerPointMatching; // 0.2675% (+7%)
 
-    mapping(uint256 => uint256) public override lastTransferTimestamp;
```

Comment:
State variables cannot be removed, ever. The `override` keyword should be fine to remove however.

Reply:
Okies.
Is there some language sugar or comment you recommend to make it clear it's no longer to be used?

Reply:
I guess use `// UNUSED; KEPT FOR UPGRADEABILITY PROXY COMPATIBILITY`